_includes/codepens/drive-standalone-upload/example.js

```diff
@@ -0,0 +1,16 @@
+var button = document.querySelector('.tinydrive-standalone-demo-upload');
+var percent = document.querySelector('.tinydrive-standalone-demo-percent');
+
+button.addEventListener('click', function () {
+  tinydrive.upload({
+    token_provider: '/your-local/jwt-provider',
+    path: '/hello',
+    name: 'hello.txt',
+    blob: new Blob('Hello world!'),
```

Comment:
This doesn't look like it'd work, as it'd need to be `new Blob(['Hello world!']),` I believe?